AmplifyPlugins/API/Sources/AWSAPIPlugin/AppSyncRealTimeClient/AppSyncRealTimeClient.swift

```diff
@@ -0,0 +1,457 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+
+import Foundation
+import Amplify
+import Combine
+@_spi(AmplifySwift) import AWSPluginsCore
```

Comment:
should the identifier be more specific? I see a few other ones named `InternalAWSPinpoint`, `LazyReference`, `PredictionsConvertRequestKind`, `KeyChainStore`, how about `Websocket` or `WebsocketClient`? We could technically move the code under the Websocket folder from AWSPluginsCore over to Amplify as public types, just something to consider, not necessarily proposing us to do that in this PR

Reply:
sure, we can use a more specific name. `WebSocket` sounds good to me.